examples/unix_socket_server/source/app.d

```diff
@@ -0,0 +1,20 @@
+import vibe.inet.url;
+import vibe.http.server;
+import vibe.http.router;
+import std.stdio;
+
+void handleHelloRequest(scope HTTPServerRequest req, scope HTTPServerResponse res)
+{
+    res.writeBody("Hello, World!", "text/plain");
+}
+
+shared static this()
+{
+	auto router = new URLRouter;
+	router.get("/hello", &handleHelloRequest);
+
+	auto settings = new HTTPServerSettings;
+	settings.bindAddresses = ["/tmp/vibe.sock"];
+
+	listenHTTP(settings, router);
```

Comment:
Have a look at the other tests for examples on how you can verify the output of the server with `requestHTTP`

Reply:
Will check that out, thanks.

Reply:
Hmm, I'm not sure how much sense it would make adding tests for this PR considering the client side (i.e. `requestHTTP`, in master since https://github.com/vibe-d/vibe.d/commit/33027eeb13c935cbcd437ba472c42cc50059ce22) for UDS doesn't have tests either.

Reply:
Would surely be good to add a test that tests both sides at once with a simple request. But since it depends on vibe-core fixes to make it work there, too, I'd defer that, though and treat this topic WIP in the meantime.